### ISSUE TITLE
Image-based tutorial

### ISSUE THREAD

**Issue by Halilibrahimaysel** [marcotcr/anchor#70]
Can you please add examples for images? I am sure it will be helpful for many people especially to see the difference between LIME and Anchors. 

Thank you.

**Comment by krishnakripaj** [comment]
I agree with you too. It will be really helpful if we can get a tutorial on images

**Comment by InonS** [comment]
See this https://github.com/marcotcr/anchor/issues/9#issuecomment-378297988
